spec/services/usps_in_person_proofing/enrollment_helper_spec.rb

```diff
@@ -13,13 +13,22 @@
   end
   let(:subject) { described_class }
   let(:subject_analytics) { FakeAnalytics.new }
+  let(:transliterator) { UspsInPersonProofing::Transliterator.new }
   let(:service_provider) { nil }
+  let(:usps_ipp_transliteration_enabled) { true }
 
   before(:each) do
     stub_request_token
     stub_request_enroll
     allow(IdentityConfig.store).to receive(:usps_mock_fallback).and_return(usps_mock_fallback)
+    allow(UspsInPersonProofing::Transliterator).to receive(:new).and_return(transliterator)
```

Comment:
in general I dislike stubbing `.new`, it's really broad. A rule I use is "don't stub what you don't own"

so a workaround would be that the enrollment helper has a memoized `transliterator` property that we stub in instead

Reply:
I tried using a memoized transliterator, and that didn't work well with tests at all. Really I prefer constructor injection (because I have had issues with both this approach and the property approach), but considered it out-of-scope to refactor for that here.

Reply:
I would still stub the `transliterator` method rather than `.new` even if we're not memoizing

Reply:
`transliterator` function stubbed. I generally prefer to avoid stubbing any part of the subject, but I'll make an exception here.